dbms/src/Flash/ResourceControl/LocalAdmissionController.cpp

```diff
@@ -0,0 +1,22 @@
+// Copyright 2023 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <Flash/ResourceControl/LocalAdmissionController.h>
+
+namespace DB
+{
+
+auto LocalAdmissionController::global_instance = std::make_unique<MockLocalAdmissionController>();
```

Comment:
So this pr only implement `MockLocalAdmissionController`?

Reply:
Yes, next pr will implement real `LocalAdmissionController`, which will communicate to GAC to fetch GAC.